website/docs/docs/use-dbt-semantic-layer/dbt-sl.md

```diff
@@ -0,0 +1,169 @@
+
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+<Snippet path="sl-prerequisites" />
+
+<Snippet path="sl-considerations-banner" />
+
+
+## Manage metrics
+
+:::info 📌
+
+New to dbt or metrics? Check out our [quickstart guide](/quickstarts) to build your first dbt project! If you'd like to define your first metrics, try our [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example project.
+
+:::
+
+If you're not sure whether to define a metric in dbt or not, ask yourself the following: 
+
+> *Is this something our teams consistently need to report on?*  
+
+An important business metric should be:
+
+- Well-defined (the definition is agreed upon throughout the entire organization)
+- Time-bound (able to be compared across time)
+
+A great example of this is **revenue** &mdash; it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand.
+
+- ✅ `Monthly recurring revenue` or `Weekly active users` or `Average order value` 
+- ❌ `1-off experimental metric`
+
+
+### Design and define metrics
+
+You can design and define your metrics in a `.yml` files nested under a metrics key in your dbt project, review the following documents: <br />
+
+- [dbt metrics](docs/build/metrics) for in-depth detail on attributes, filters, how to define and query your metrics, and [dbt-metrics package](https://github.com/dbt-labs/dbt_metrics)
+- [dbt Semantic Layer quickstart](/docs/use-dbt-semantic-layer/quickstart-semantic-layer) to get started
+
+## Related questions
+
+<details>
+  <summary>How do I migrate from the legacy Semantic Layer to the new one?</summary>
+  <div>
+    <div>If you're using the legacy Semantic Layer, we highly recommend you <a href="https://docs.getdbt.com/docs/dbt-versions/upgrade-core-in-cloud">upgrade your dbt version </a> to dbt v1.6 or higher to use the new dbt Semantic Layer. Refer to the dedicated <a href="https://docs.getdbt.com/guides/migration/sl-migration"> migration guide</a> for more info.</div>
+  </div>
+</details>
+    
+<details>
+  <summary>How are you storing my data?</summary>
+  <div>
+    <div>The dbt Semantic Layer does not store, or cache, or log your data. On each query to the Semantic Layer, the resulting data passes through dbt Cloud servers where it is never stored, cached, or logged. The data from your data platform gets routed through dbt Cloud servers, to your connecting data tool.</div>
+  </div>
+</details>
+<details>
+  <summary>Is the dbt Semantic Layer open source?</summary>
+  <div>
+    <div>Some components of the dbt Semantic Layer are open source like dbt-core, the dbt_metrics package, and the BSL-licensed dbt-server. The dbt Proxy Server (what is actually compiling the dbt code) and the Discovery API are not open sources. <br></br><br></br>
+      
+During Public Preview, the dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise).<br></br><br></br>
```

Comment:
this sentence and the unordered list below it seems _out of place_ for "Is the dbt Semantic Layer open source?" suggest moving the info elsewhere